### PR TITLE
build: no longer need Python 3.5 to build illumos-gate

### PR DESCRIPTION
**Pull request by Bill-Sommerfeld**

### CODE REVIEW

**Review by mtelka — APPROVED** [review]
lgtm

**Review by tsoome — APPROVED** [review]

File: docs/developers/build.md [hunk 1]
```diff
@@ -54,13 +54,7 @@ chosen:
 ### OpenIndiana
 
 On OpenIndiana, the `build-essential` package includes the GCC compilers
-and other tools required to build illumos.  In addition, for now, we
-should install the older Python 3.5 package for a complete build.
-
-```
-sudo pkg install build-essential \
-    runtime/python-35
-```
```
Comment (jclulow) [inline docs/developers/build.md:63]: I think we should leave the command to install the package here, just with the amended list of packages?

Comment (Bill-Sommerfeld) [reply docs/developers/build.md]: Agreed.   Rebased and changed as requested.

**Comment by jclulow** [comment]
Thanks.  Deployed!

**Pull request merged**